### ISSUE TITLE
Sites on port 8000 - 8080

### ISSUE THREAD

**Issue by ozzymanborn** [krlvm/PowerTunnel#25]
If a blocked site (mostly iptv sites using that) has a main site in 80 port but iptv (or playlist) server on 8000 or 8080 port powertunnel works good on 80 port site/dashboard but " Hmmm… can't reach this page / The connection was reset ." error happens on port 8000 (or 8080) site.

I have same issue on windows goodbyedpi. I hope it can be implemented somehow because It would also solve some internet radio streams not opening when powertunnel on...

**Comment by krlvm** [comment]
Are you able to open non-blocked site on this port range?

**Comment by ozzymanborn** [comment]
Yes when I tried to load a playlist from non blocked 8080 or 8000 site it's work. (I tried on kodi and it's getting playlist on port 8000 or 8080) 

Normal behaviour without entering user or password is http://*****:8000 is access denied writing top on the page on browsers on android. when the connection was reset writing happens I'm understand some block.

**Comment by ozzymanborn** [comment]
On windows goodbyedpi it solves with 
start "" goodbyedpi.exe -1 -w --port 8080 --port 8000
if it can be implementable to powertunnel it could be fixed...
